CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.5.0 (2020-01-27)
+
+- Added the inline delete component. [#56](https://github.com/blackbaud/skyux-layout/pull/56)
+- Added logic to the card component to handle the inline delete component. [#56](https://github.com/blackbaud/skyux-layout/pull/56)
```

Comment:
`to handle`? Could we be more specific? Maybe something about allowing the inline delete component to appear inside the cards template? Not sure how to word this.

Reply:
How about `Added logic to the card component to allow it to contain the inline delete component.`? Does that sound better to you, @Blackbaud-AlexKingman?

Reply:
Hmmmm. I think I like the second one better @blackbaud-johnly

Reply:
Sounds good. ... I'll make the change.